flycheck.el

```diff
@@ -6390,6 +6391,14 @@ See URL `http://www.jscs.info'."
                    (flycheck-sanitize-errors errors)))
   :modes (js-mode js2-mode js3-mode))
 
+(flycheck-define-checker javascript-standard
+  "A code style linter for the JavaScript Standard Style.
+
+See URL `https://github.com/feross/standard'."
+  :command ("standard" source)
```

Comment:
That's sad :(  Would you mind to ask the `standard` developers whether they could add an option for checkstyle XML output?  eslint supports it, so it shouldn't be too difficult for them, hopefully.

Doesn't stand against merging, though:  I'll merge it with patterns, too, but checkstyle would be nice to have, since it's less fragile.